### ISSUE TITLE
Remove Geocode Lookup

### ISSUE THREAD

**Issue by codecakes** [Xcov19/mycovidconnect#20]
- [ ] Pull the branch: first-timers-only-remove-google-geocode-lookup
- [ ] [Remove the location lookup ](https://github.com/Xcov19/mycovidconnect/blob/dbb5a23465403ca19cc7b50030a838b6770dbe18/src/firebase/fireStore.js#L65) from L65 to L81
- [ ] Reuse from L82 querySnapshotResults instead of `results`

**Comment by CarlKarlQarl** [comment]
Hi there, I'm a first timer, and I wanted to take a shot at this. Let me know if I miss something. Thanks!

**Comment by codecakes** [comment]
@CarlKarlQarl yup i think this is u https://github.com/Xcov19/mycovidconnect/pull/25
